src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/DiscountProgramAdministratorResponse.cs

```diff
@@ -6,7 +6,7 @@ namespace Exadel.CrazyPrice.Common
     /// <summary>
     /// Gets full information for an Administrator.
     /// </summary>
-    public class DiscountProgramWithDateModify
+    public class DiscountProgramAdministratorResponse
```

Comment:
Please check that this separate classes will be needed. Probably we can use same class with empty fields and setup rules to serialize to JSON with empty fields

Reply:
It fixed [https://github.com/yegorl/exadel_discounts_be/pull/3/commits/842b9c9fd0ced13302bc339c977307d14e491896](url)